test/built-ins/RegExp/prototype/Symbol.replace/y-set-lastindex.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Sets the `lastIndex` property to the index of the first match
```

Comment:
Like in the file for **Symbol.match**, it should say "end index" rather than "index". We should also update other files.